### PR TITLE
Lib package

### PR DESCRIPTION
**Pull request by jan-schutte**
Hi Andre,
yaml files, renamed datasources
renamed Nox.Microservice to Nox.Lib

### CODE REVIEW

**Review by andresharpe — COMMENTED** [review]
Mis-spelling. Bunny <> Bunnty

Your code needs to pass a "goo init" before submission ;)

File: design/workplace.service.nox.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ Database:
   Password: Developer*123
 
 MessagingProviders:
-  - Name: Hasie
+  - Name: Bunnty
```
Comment (andresharpe) [inline design/workplace.service.nox.yaml]: Spelling error. Did you test with "goo init"?

Comment (jan-schutte) [reply design/workplace.service.nox.yaml]: Fixed

**Review by andresharpe — APPROVED** [review]
Good to go

**Pull request merged**